packages/web-components/src/globals/mixins/stable-selector.ts

```diff
@@ -44,6 +44,13 @@ const StableSelectorMixin = <T extends Constructor<HTMLElement>>(Base: T) => {
         .constructor as typeof StableSelectorMixinImpl;
       if (stableSelector) {
         this.dataset.autoid = stableSelector;
+
+        const css = `${this.tagName.toLowerCase()}:not(:defined) { display: none; }`;
+        const head = document.head || document.getElementsByTagName('head')[0];
+        const style = head.querySelector("style") || document.createElement('style');
```

Comment:
```suggestion
        const style = head.querySelector('style') || document.createElement('style');
```

Super small thing but after that lgtm!

Reply:
Looks like you need to run a `yarn format` too.